python/rikai/spark/sql/codegen/pytorch.py

```diff
@@ -63,6 +65,7 @@ def torch_inference_udf(
                     dataset,
                     batch_size=batch_size,
                     num_workers=num_workers,
+                    pin_memory=pin_memory,
                 ):
                     batch = batch.to(device)
```

Comment:
I *think* if pin_memory is set to be true, you also need to do `batch.to(device, non_blocking=True)` for performance gains.